test/integration/metacluster_test.go

```diff
@@ -47,7 +47,4 @@ var _ = Describe("Metacluster", func() {
 	AfterEach(func() {
 		kindCluster.Teardown()
 	})
-
-	// TODO: validate that the controller-manager is deployed and the
```

Comment:
**resolved**: this test suite is empty and not being used. @justaugustus to confirm there is nothing to do here.

Reply:
Fine to remove